grouper/repositories/permission_grant.py

```diff
@@ -55,6 +55,22 @@ def service_account_grants_for_permission(self, name):
         # type: (str) -> List[ServiceAccountPermissionGrant]
         return self.repository.service_account_grants_for_permission(name)
 
+    def permission_grants_for_service_account(self, name):
+        # type: (str) -> List[ServiceAccountPermissionGrant]
+        user_details = self.graph.get_user_details(name)
+        permissions = []
+        for permission_data in user_details["permissions"]:
+            permission = ServiceAccountPermissionGrant(
+                service_account=name,
+                permission=permission_data["permission"],
+                argument=permission_data["argument"],
+                granted_on=datetime.utcfromtimestamp(permission_data["granted_on"]),
+                is_alias=permission_data["alias"],
+                grant_id=None,
```

Comment:
I feel like I asked this on a previous PR, but do we have a sense for when/how `grant_id` is necessary and why it's okay to not set it if we're using a graph-backed repo?

Reply:
It's used by the frontend to uniquely identify a grant, since otherwise it has to be represented by the tuple of (group, permission, argument) or (service account, permission, argument), which is painful to work with in URLs. I think we'll eventually need to store it in the graph as well (probably by caching the full `ServiceAccountPermissionGrant` objects in the graph). This is just a temporary workaround to avoid doing too much work now.